tests/unit/lib/charms/sdcore_upf/v0/test_fiveg_n3_requirer.py

```diff
@@ -27,7 +27,7 @@ def test_given_relation_with_n3_profider_when_fiveg_n3_available_event_then_n3_i
 
         self.harness.update_relation_data(
             relation_id=relation_id,
-            app_or_unit="whatever-app/0",
+            app_or_unit="whatever-app",
```

Comment:
The test name has a typo:
profider -> provider
test_given_relation_with_n3_provider_when_fiveg_n3_available_event_then_n3_information_is_provided

Reply:
Good catch :) 
Fixed.